sdk/include/ndk/iotypes.h

```diff
@@ -826,28 +826,39 @@ typedef struct _FILE_TRACKING_INFORMATION
 //
 // File System Information structures for NtQueryInformationFile
 //
+#ifndef FILE_FS_DEVICE_INFORMATION_DEFINED
+#define FILE_FS_DEVICE_INFORMATION_DEFINED
```

Comment:
Why are these `..._DEFINED` needed in the NDK? The NDK shouldn't be included in wine code. Or, you've now added wine includes in non-wine code, or added unnecessary includes that are redundant with the ndk.

Reply:
Deleted `#ifndef ..._DEFINED` etc. Build is broken now.